library/libwaku.nim

```diff
@@ -84,6 +85,31 @@ proc onReceivedMessage(ctx: ptr WakuContext): WakuRelayHandler =
           RET_ERR, unsafeAddr msg[0], cast[csize_t](len(msg)), ctx[].eventUserData
         )
 
+proc onTopicHealthChange(ctx: ptr WakuContext): TopicHealthChangeHandler =
```

Comment:
We are using a similar approach in other place. The very next time we add another similar `proc` we might need to refactor to avoid duplicate code :)

Reply:
100%! I agree that for the next app callback we should turn this into a template :))